packages/webdriverio/src/commands/browser/reloadSession.js

```diff
@@ -23,9 +23,9 @@ export default async function reloadSession () {
     const oldSessionId = this.sessionId
 
     /**
-     * end current running session
+     * end current running session, if session already gone suppress exceptions
      */
-    await this.deleteSession()
+    try { await this.deleteSession() } catch (err) { /** ignoring all exceptions caused by deleteSession, we may want to reloadSession even if remote session has died */ }
```

Comment:
May you please use newlines here

Reply:
Of course! I was trying to make it a one-liner but totally understand! Thanks will make changes now.

Reply:
Fixed, we may think about moving this into `deleteSession` in the future as perhaps we should always resolve in that Promise.

Reply:
```
export default async function deleteSession () {
    await this.browser.close()
    return null
}
```

I could move it into here if you think it is a better position for it.